### PR TITLE
Add command liveness probe support

### PR DESCRIPTION
**Pull request by lonefreak**
The changes will allow to add a command liveness probe to a container. You can specify:

command to execute (does not support httpGet) - required field
timeout seconds
initial delay seconds
failure threshold
period seconds
success threshold

### CODE REVIEW

**Comment by carlossg** [comment]
you could have force pushed to the same PR, now all the comments are lost in the other one #125 :(

**Comment by lonefreak** [comment]
Sorry for the delay, I was on vacation. True, I should have done that. Should I still do it (push force and reopen the other PR) or will you simply keep the old one pointing here as a reference? 

Anyway I tried to follow all the comments that are in the old PR  #(125). Is there anything else to fix?

File: src/main/resources/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate/config.jelly [hunk 1]
```diff
@@ -39,6 +39,14 @@
                                   deleteCaption="Delete Environment Variable" />
     </f:entry>
 
+  <f:entry field="livenessProbe" title="${%Liveness Probe}">
```
Comment (carlossg) [inline src/main/resources/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate/config.jelly]: liveness probe should be in the advanced section, at the end
it's also showing "Liveness Probe > Exec Action should not be empty"

Comment (lonefreak) [inline src/main/resources/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate/config.jelly]: Ah, ok. I will fix this soon and submit a PR. Thanks for the good feedback :)

**Pull request merged**